mypy_django_plugin/transformers/auth.py

```diff
@@ -10,17 +10,18 @@
 from mypy_django_plugin.lib import fullnames, helpers
 
 
+def _get_abstract_base_user(api: SemanticAnalyzer) -> ProperType:
+    sym = api.lookup_fully_qualified(fullnames.ABSTRACT_BASE_USER_MODEL_FULLNAME)
+    assert isinstance(sym.node, TypeInfo)
+    return fill_typevars_with_any(sym.node)
+
+
 def get_user_model(ctx: AnalyzeTypeContext, django_context: DjangoContext) -> MypyType:
     assert isinstance(ctx.api, TypeAnalyser)
     assert isinstance(ctx.api.api, SemanticAnalyzer)
 
-    def get_abstract_base_user(api: SemanticAnalyzer) -> ProperType:
```

Comment:
Creating function inside function is rather expensive and we pay the function creation each time the outer function is called, this was not required here since the inner func does not use closures